### PR TITLE
Merge hyperboria:master

### PR DESCRIPTION
**Pull request by lgierth**
Mostly documentation changes, and:

- contrib: remove bash/peers.sh
- build: allow `-O0`
- tools: fix timeout typo in cjdnsadmin.js

@dangowrt's commits are already applied in crashey

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+### Cryptographic verification and routing
 
-Anything you can submit that gets us closer to understanding some bug or function within cjdns is welcome. By starting an inquiry, you make it just a little bit easier for someone else to continue it. With that in mind, please read [this note on reporting bugs](bugs/reporting.md) and [our more general contribution policy](bugs/policy.md). Get involved!
+Establishing a link between two nodes includes a cryptographic authorization process, after which other nodes within the network can establish a connection with the new node.
 
-## License
+Due to the relationship between each node's public key and its ipv6, and the fact that this relationship is verified upon connecting to a node, a user can be sure that if it connects to an IPv6 address and receives a response then the node that responded possesses the requisite private key to decrypt the response.
 
+<<<<<<< HEAD
```
Comment (kpcyrd) [inline doc/README.md]: Something went wrong here

File: doc/configure.md [hunk 2]
```diff
@@ -192,7 +201,7 @@ IP Tunneling
 
 IP Tunneling will allow you to connect from the cjdns network to another outside network. This is still a work in-progress; although it does function, it requires a bit of manual configuration on both ends to make it useful.
 ````javascript
-        // System for tunneling IPv4 and ICANN IPv6 through cjdns.
+        // System for tunneling IPv4 and ICANN IPv6 through cjdn which will eventually be merged to master..
```
Comment (kpcyrd) [inline doc/configure.md]: Wasn't this in master already? (Also, typo)

Comment (lgierth) [inline doc/configure.md]: Yeah no idea. There's a lot of crap in the docs repo -- we need to be more serious about it. Stuff like the Emercoin repo is just irelevant. Also we should just ruthlessly delete anything that's only slightly outdated.

File: doc/README.md (hunk 1 above)
Comment (lgierth) [inline doc/README.md]: Hrm, right, fixed.

**Pull request merged**